api/src/test/resources/OML_O21.hl7

```diff
@@ -1,4 +1,4 @@
 MSH|^!\&|iSantePlus||OpenELIS||20180330150703.345+{time_offset}00||OML^O21^OML_O21|2018033015070300000|D|2.5.1
```

Comment:
According to Inbound (HL7 2.5.1 OML^O21)
section 3rd 'Sending Application'=iSantePlus
section 5th Sending Facility'=OpenELIS
are not required.

Reply:
Same as above